### DISCUSSION TITLE
Feature Request: add view counter / hit counter / page views count

### DISCUSSION THREAD

**Discussion by ElectricRCAircraftGuy** [mmistakes/minimal-mistakes#2715] [Ideas]
## Summary

I'd really like a way to track the number of unique hits, or page views, on my site. 

## Motivation

This helps us know if our content is being seen. If the page view counter is publicly visible, it helps others know if our content is being seen, adding a sense of "legitimacy" to our site. 

Knowing which of our blog posts are being seen the most also gives us a sense of what our audience wants and likes, and helps us to know how to tailor our content better. 

## Drawbacks

Adding hit counters may require separate servers or possibly even paid services. They may be hard to implement on static sites. 

I'm not sure how to add a hit counter, but here's some research I've done to help get you started: https://github.com/jhvanderschee/jekyllcodex/issues/14.

Many thanks in advance. I really think this would be a great addition to have!

**Comment by mmistakes** [comment]
> I'd really like a way to track the number of unique hits, or page views, on my site.

You can achieve this by setting up one of the integrated analytics providers e.g. Google Analytics. Consult the documentation. https://mmistakes.github.io/minimal-mistakes/docs/configuration/#analytics

If you want to add a hit counter you'll need to customize `_includes/analytics-providers/custom.html`.
